src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -67,7 +67,7 @@ public static void Main(string[] args)
                 {
                     debugArgPassed = true;
                 }
-                else if (arg.StartsWith("--trace:"))
+                else if (arg.StartsWith("--trace="))
```

Comment:
I wish this were not so fragile. Should we add a test that guarantees all args created for the agent process use this form?

Reply:
This one broke due to the method to write the arguments being standardised. I feel like now there's a standard method rather than each one being added individually, it will hopefully be more robust?

We might want to build on that when we start making agents-in-extensions, of course.